### PR TITLE
ui searchview and function 1

### PR DESCRIPTION
**Pull request by datchiz**

### CODE REVIEW

**Comment by datchiz** [comment]
function em mới viết 1 nửa

File: app/src/main/java/com/framgia/fdms/screen/listDevice/ListDeviceContract.java [hunk 1]
```diff
@@ -24,12 +26,24 @@
         void hideProgressbar();
 
         void onRegisterDeviceClick();
+
+        void DeviceCategoryLoaded(List<Category> categories);
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fdms/screen/listDevice/ListDeviceContract.java]: onCategoryLoaded. onDeviceStatusLoaded

File: app/src/main/res/layout/fragment_listdevice.xml [hunk 2]
```diff
@@ -16,9 +16,50 @@
...
+            android:layout_below="@id/searchview"
+            android:orientation="horizontal"
+            android:weightSum="2"
+            >
+
+            <Button
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:hint="Status"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/fragment_listdevice.xml]: hardcode

File: app/src/main/res/layout/fragment_listdevice.xml [hunk 3]
```diff
@@ -16,9 +16,50 @@
...
+                android:onClick="@{()-> viewModel.onChooseStatus()}"
+                android:text="@={viewModel.status.name}"
+                android:textSize="@dimen/sp_8"
+                />
+
+            <Button
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:hint="Category"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/fragment_listdevice.xml]: hardcode

**Pull request merged**